src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -545,11 +545,14 @@ private Reports CreateReports(bool verbose, bool quiet)
             return reports;
         }
 
-        private static string GetVersion()
+        private string GetVersion()
```

Comment:
Btw, I need that extension method in the code that I'm writing now so it would be great if you can provide it :)